pkg/cmd/generate-postgres-metadata-tables/main.go

```diff
@@ -28,17 +28,36 @@ import (
 	"os"
 
 	"github.com/cockroachdb/cockroach/pkg/sql"
+	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/jackc/pgx"
+	"github.com/lib/pq/oid"
 )
 
 const getServerVersion = `SELECT current_setting('server_version');`
 
+// Flags or command line arguments.
 var (
 	postgresAddr   = flag.String("addr", "localhost:5432", "Postgres server address")
 	postgresUser   = flag.String("user", "postgres", "Postgres user")
 	postgresSchema = flag.String("catalog", "pg_catalog", "Catalog or namespace, default: pg_catalog")
 )
 
+// Map for unimplemented types.
+var unimplementedEquivalencies = map[oid.Oid]oid.Oid {
+	// These types only exists in information_schema
+	oid.Oid(13438):     oid.T_int4,        // cardinal_number
+	oid.Oid(13450):     oid.T_bool,        // yes_or_no
+	oid.Oid(13441):     oid.T_text,        // character_data
+	oid.Oid(13443):     oid.T_text,        // sql_identifier
+	oid.Oid(13448):     oid.T_timestamptz, // time_stamp
+
+	// Other types
```

Comment:
this is another good reference: https://www.npgsql.org/doc/dev/type-representations.html

(we could even add this link the comments)

Reply:
Done.